conans/client/graph/graph_builder.py

```diff
@@ -76,7 +78,7 @@ def _load_deps(self, node, down_reqs, dep_graph, public_deps, down_ref, down_opt
         param down_ref: ConanFileReference of who is depending on current node for this expansion
         """
         # basic node configuration
-        new_reqs, new_options = self._config_node(node, down_reqs, down_ref, down_options)
+        new_reqs, new_options = self._config_node(node, down_reqs, down_ref, down_options, aliased)
 
         self._resolve_deps(node, aliased, update, remote_name)
```

Comment:
If `_config_node` performs the alias substitution, then it is no longer needed in `_resolve_deps`.

Reply:
I have tried a few things, and tests are broken. Leaving it as-is.